src/controllers/order.controllers.js

```diff
@@ -0,0 +1,24 @@
+const postOrderService = require('../services/order.service');
+
+const postOrder = async (req,res) => {
```

Comment:
```suggestion
const createOrder = async (req,res) => {
```
C. R. U. D는 create, get, update, delete에 맞춰서 통일되게 작성해주세요~